Source/IntegrationTests/TestFiles/LitTests/LitTest/verification/filter.dfy

```diff
@@ -1,15 +1,14 @@
-// RUN: %verify --filter-position='blaergaerga' %s > %t
-// RUN: %verify --filter-position='C:\windows\path.dfy' %s >> %t
-// RUN: ! %verify --filter-position='filter.dfy' %s >> %t
-// RUN: ! %verify --filter-position='filter.dfy:14' %s >> %t
+// RUN: %verify --filter-position='C:\windows\path.dfy' %s > %t
 // RUN: ! %verify --filter-position='src/source1.dfy:5' %S/Inputs/dfyconfig.toml >> %t
-// RUN: %verify --filter-position='src/source1.dfy:2' %S/Inputs/dfyconfig.toml >> %t
-// RUN: %diff "%s.expect" "%t"
-
-method Succeeds()
-  ensures true {
-}
-
-method Fails() 
-  ensures false {
-}
+// RUN: %verify --filter-position='src/source1.dfy:1' %S/Inputs/dfyconfig.toml >> %t
+// RUN: ! %verify --filter-position='e.dfy' %S/inputs/single-file.dfy >> %t
+// RUN: %verify --filter-position='e.dfy:2' %S/inputs/single-file.dfy >> %t
+// RUN: %verify --filter-position='blaergaerga' %S/inputs/single-file.dfy >> %t
```

Comment:
Should it raise an error or at least a warning if the position does not match a given regex?

Reply:
I'd rather not. I feel it's not worth the effort/maintenance because that only catches one problematic case. This command outputs `Dafny program verifier finished with 0 verified, 0 errors` which the user should always inspect when using filter to see if they're matching on what they intended too.